### ISSUE TITLE
Add a measuring spoon icon

### ISSUE THREAD

**Issue by zbigniewzolnierowicz** [lucide-icons/lucide#2320]
### Icon name

Spoon

### Use cases

I'm developing a recipe app, and am using icons for different measurements, i.e.: a cup for cups, scales for grams/kilograms, and a spoon for tea/tablespoons. This would be very useful for the food industry,  or for medicine - to visually describe, i.e.: a (cough) syrup.

### Design ideas

![image](https://github.com/user-attachments/assets/6dd2a6b8-0c78-40bf-b50e-9ffc066c58a9)
Quick and dirty mockup I whipped up in Inkscape.

For alternative designs, look at the one in [MingCute](https://github.com/Richard9394/MingCute/blob/main/svg/food/spoon_line.svg)

### Checklist

- [X] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)
- [X] I have searched existing icons to make sure it does not already exist and I didn't find any.
- [X] I am not requesting a brand logo and the art is not protected by copyright.
- [X] I am not requesting an icon that includes religious, political imagery or hate symbols.
- [X] I have provided appropriate use cases for the icon(s) requested.

**Comment by jamiemlaw** [comment]
In a similar style to `utensils` and `utensils-crossed`:

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M12+11v11%22+/%3E%3Cellipse+cx=%2212%22+cy=%226.5%22+rx=%223%22+ry=%224.5%22+/%3E"><img alt="icons" width="200px" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMTIgMTF2MTEiIC8+CiAgPGVsbGlwc2UgY3g9IjEyIiBjeT0iNi41IiByeD0iMyIgcnk9IjQuNSIgLz4KPC9zdmc+.svg?utm_source=lucide-studio&utm_medium=embed?utm_source=lucide-studio&utm_medium=embed"/><br/>Open lucide studio</a>

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M10.174+10.174+22+22%22+/%3E%3Cpath+d=%22M5+2a3.674+5.196-45+0+1+3+9+3.674+5.196-45+0+1-3-9%22+/%3E"><img alt="icons" width="200px" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMTAuMTc0IDEwLjE3NCAyMiAyMiIgLz4KICA8cGF0aCBkPSJNNSAyYTMuNjc0IDUuMTk2LTQ1IDAgMSAzIDkgMy42NzQgNS4xOTYtNDUgMCAxLTMtOSIgLz4KPC9zdmc+.svg?utm_source=lucide-studio&utm_medium=embed?utm_source=lucide-studio&utm_medium=embed"/><br/>Open lucide studio</a>

Technically regular spoons, not measuring spoons, but these could conceivably have more use cases.

**Comment by zbigniewzolnierowicz** [comment]
Yeah, I think the "measuring" part doesn't need to be there - just a generic spoon is good, and this is very good - especially the latter.
